src/model/mod.rs

```diff
@@ -68,3 +69,7 @@ impl Display for GameVersion {
         }
     }
 }
+
+pub trait FromRobTopStr {
+    fn from_robtop_str(robtop_str: &str) -> Result<Self, DeError> where Self: Sized;
+}
```

Comment:
what's this trait for?

Reply:
Leftover testing while I was trying to figure out the logic of the build script, I'll remove it in another commit.